demo/src/app/components/datepicker/demos/basic/datepicker-basic.ts

```diff
@@ -0,0 +1,19 @@
+import {Component} from '@angular/core';
+
+const now = new Date();
+
+@Component({
+  selector: 'ngbd-datepicker-basic',
+  template: require('./datepicker-basic.html')
```

Comment:
Oh, it's the script that generates the demo skeleton apparently. Good catch, thanks. Will update the script in a separate PR.